docs/en/Admin-CLI.md

```diff
@@ -41,12 +41,18 @@ UFS URI like `hdfs://<name-service>/`, and not `hdfs://<name-service>/<folder>`.
 
 The `report` command provides Alluxio running cluster information. 
 
-For example, `report` can be used to get runing cluster state and capacity information.
+For example, `report` can be used to get summarized cluster information and capacity information.
```

Comment:
since we immediately give examples and include comments to describe them, I don't think we need this sentence any more

Reply:
Yes, it is a duplicated info. Deleted, thanks